app/javascript/components/editor-config/keys.js

```diff
@@ -62,12 +62,9 @@ const keys = {
   'Mod-]': sinkListItem(schema.nodes.list_item),
   'Shift-Ctrl-0': setBlockType(schema.nodes.paragraph),
   'Shift-Ctrl-\\': setBlockType(schema.nodes.code_block),
```

Comment:
This removes that key for both editors? Is that intended? I think we should be preserving `shift-ctrl-1`.

Reply:
> This removes that key for both editors? Is that intended?

My intention is to remove `h1` keys from both the title editor and body editor.

Note in a separate PR https://github.com/jellypbc/poster/pull/315 the title editor is given a separate set of keys. The keys in this file are specific to the body editor. 

> are we trying to treat the page with just one h1, which is the title editor?

This is my intention, yes.

> Or what if pages end up being parsed from grobid with h1, and we shift all headings down one?

When we parse PDFs with grobid, I would like to see the biggest heading be `h2`. 

@dennydennydenny would you still like me to put `h1` keys back for the body editor?

Reply:
the title editor is given a limited set of keys in https://github.com/jellypbc/poster/pull/316

```
const titleKeys = {
  'Mod-z': undo,
  'Shift-Mod-z': redo,
  'Mod-y': redo,
  'Mod-i': toggleMark(schema.marks.em),
  'Mod-Shift-m': addAnnotation,
}
```

Reply:
okidoke :+1: